migrations/2020-09-17-130740_create_receipts/up.sql

```diff
@@ -0,0 +1,53 @@
+CREATE TYPE receipt_type AS ENUM ('ACTION', 'DATA');
+CREATE TYPE action_type AS ENUM (
+    'CREATE_ACCOUNT',
+    'DEPLOY_CONTRACT',
+    'FUNCTION_CALL',
+    'TRANSFER',
+    'STAKE',
+    'ADD_KEY',
+    'DELETE_KEY',
+    'DELETE_ACCOUNT'
+);
+
+CREATE TABLE receipts (
+    receipt_id text PRIMARY KEY,
+    block_height numeric(45, 0), -- numeric(precision) 45 digits should be enough to store u128::MAX
+--     chunk_hash bytea NOT NULL,
+    predecessor_id text NOT NULL,
+    receiver_id text NOT NULL,
+    receipt_kind receipt_type NOT NULL,
+    CONSTRAINT block_receipts_fk FOREIGN KEY (block_height) REFERENCES blocks(height) ON DELETE CASCADE
+--     CONSTRAINT chunk_receipts_fk FOREIGN KEY (chunk_hash) REFERENCES chunks(hash) ON DELETE CASCADE
```

Comment:
why not use the chunk constraint?

Reply:
`StreamerMessage.local_receipts` don't belong to any chunk yet, they don't have the relation, so this constraint will always fail. I'm going to deal with them in this PR as well, it is temporary.